Microsoft.Toolkit.Uwp.UI.Controls/BladeControl/BladeControl.Properties.cs

```diff
@@ -37,6 +37,11 @@ public partial class BladeControl
         public static readonly DependencyProperty ToggleBladeProperty = DependencyProperty.RegisterAttached(nameof(ToggleBlade), typeof(string), typeof(BladeControl), new PropertyMetadata(null));
 
         /// <summary>
+        /// Identifies the <see cref="FullScreenBlades"/> attached property.
+        /// </summary>
+        public static readonly DependencyProperty FullScreenBladesProperty = DependencyProperty.RegisterAttached(nameof(FullScreenBlades), typeof(bool), typeof(BladeControl), new PropertyMetadata(false));
```

Comment:
Have a PropertyChangedCallback when the value changes. Within it check if the value is true. 
If true: change the width/height of the items and cache the previous values (can be done in a dictionary) and change the `HorizontalSnapPointsType` of the ScrollViewer to be `MandatorySingle`. 
If false: Set the width/height back to what they were previously and set `HorizontalSnapPointsType` to `Optional`